charts/prometheus-snmp-exporter/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Prometheus SNMP Exporter
 name: prometheus-snmp-exporter
-version: 0.1.5
+version: 0.1.6
```

new feature should raise minor version.
```suggestion
version: 0.2.0
```